php/Asset.php

```diff
@@ -119,4 +95,38 @@ private function get_script_config( $slug ) {
 		$plugin_path = $this->plugin->get_dir();
 		return require "{$plugin_path}/js/dist/{$slug}.asset.php";
 	}
+
+	/**
+	 * Registers the block.
+	 */
+	public function register_block() {
```

Comment:
An alternative to this is using a `blocks.json` file Genesis Custom Blocks [allows](https://developer.wpengine.com/genesis-custom-blocks/get-started/export-custom-blocks-to-include-in-your-theme/).

You can define all of the blocks you want that way.